proto/arkeo/claim/query.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package arkeonetwork.arkeo.claim;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "arkeo/claim/params.proto";
+
+option go_package = "github.com/arkeonetwork/arkeo/x/claim/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  
+  // Parameters queries the parameters of the module.
+  rpc Params (QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get = "/arkeonetwork/arkeo/claim/params";
```

Comment:
Question: do we really want to have `arkeonetwork` in the url?  will `arkeo` be good enough?

Reply:
[49669cf](https://github.com/arkeonetwork/arkeo/pull/29/commits/49669cf42e54ff5da8282c5e785d24e44f85b4fa)

Reply:
I think this comes down to do we want to stray away from defaults or not? ignite scaffold is going to default to the fully qualified url and naming convention.  I don't have a strong preference, but we will have to change everything we scaffold in the future for consistency.